Formula/mongoose.rb

```diff
@@ -3,7 +3,7 @@ class Mongoose < Formula
   homepage "https://github.com/cesanta/mongoose"
   url "https://github.com/cesanta/mongoose/archive/6.17.tar.gz"
   sha256 "5bff3cc70bb2248cf87d06a3543f120f3b29b9368d25a7715443cb10612987cc"
-  license any_of: ["GPL-2.0", "Cesanta"]
+  license any_of: ["GPL-2.0-only"]
```

Comment:
Sorry, maybe I wasn't clear enough here. No quotes or brackets needed:

```suggestion
  license "GPL-2.0-only"
```

Reply:
Oops! That was definitely an overlook on my side — it was late here 😅 I'll update.